src/hb-ot-layout-gpos-table.hh

```diff
@@ -675,22 +669,14 @@ struct SinglePos
     unsigned subset_format = 1;
     hb_array_t<const Value> first_val_iter = hb_second (*glyph_val_iter_pairs);
 
-    + glyph_val_iter_pairs
-    | hb_map (hb_second)
-    | hb_apply ([&] (hb_array_t<const Value> val_iter)
-		{
-		  + hb_zip (val_iter, first_val_iter)
-		  | hb_apply ([&] (const hb_pair_t<Value, Value>& _)
-			      {
-				if (_.first != _.second)
-				{
-				  subset_format = 2;
-				  return;
-				}
-			      })
-		  ;
-		})
-    ;
+    for (hb_array_t<const Value> val_iter : + glyph_val_iter_pairs
+					    | hb_map (hb_second))
+      for (const hb_pair_t<Value, Value> _ : hb_zip (val_iter, first_val_iter))
+	if (_.first != _.second)
+	{
+	  subset_format = 2;
+	  continue;
```

Comment:
Why continue?  Should just return immediately.

Reply:
Done later